src/tacacs/bash/bash_tacplus/configure.ac

```diff
@@ -0,0 +1,71 @@
+dnl
+dnl File:        configure.in
```

Comment:
>dnl  [](http://example.com/codeflow?start=0&length=4)

What is "dnl"?

Reply:
This .ac file is a autoreconf template, so dnl is a macro for autoreconf.